doc/source/rllib-env.rst

```diff
@@ -203,6 +203,29 @@ Here is a simple `example training script <https://github.com/ray-project/ray/bl
 
 To scale to hundreds of agents, MultiAgentEnv batches policy evaluations across multiple agents internally. It can also be auto-vectorized by setting ``num_envs_per_worker > 1``.
 
+
+PettingZoo Multi-Agent Environments
```

Comment:
@sven1977 how should we document the env integrations? I was trying to find how we document DMLab / Unity etc. integration but I only see the examples link.